today-utilities.php

```diff
@@ -79,6 +79,14 @@ function tu_init() {
 		// Add options pages.
 		add_action( 'init', 'tu_add_gmucf_options_page' );
 		add_action( 'init', 'tu_add_main_site_feed_options_page' );
+
+		// Register ACF fields for CPTs, Taxonomies, and Option pages
+		if ( is_plugin_active( 'advanced-custom-fields-pro/acf.php' ) ) {
```

Comment:
I know we've had issues in the past using `is_plugin_active()`--in particular, it requires `wp-admin/includes/plugin.php` to be included to be used outside of admin views.  We may want to consider doing a `class_exists( 'acf_pro' )` check instead.